fix the mulgore priest quest 27066 and sql file

cnmir2 opened the pull request:
frank i fix the  mulgore priest quest 27066 and test ok and want to pr to the master  please check :)

cnmir2 committed:
FIX THE QUEST 27066 MULGORE priest quest

cnmir2 committed:
20170121quest27066

gpn39f commented on src/server/scripts/Kalimdor/zone_mulgore.cpp:
```diff
@@ -428,7 +428,47 @@ class npc_agitated_earth_spirit_36845 : public CreatureScript
+{
+	NPC_WOUNDED_BRAVE = 45199,
+	SPELL_FLASH_HEAL_2061 = 2061,
+	NPC_KILLCREDIT = 44175,
```
if i look on wiki: there are 2 quest that work with this npc.. quest 27066 and quest 27067
so the brave npc must accept both spell for KillMonsterKredit
spell 2061 flash heal priest
spell 774 Rejuvenating druid
(i hope i have seen it correct.. many quest have changed metween 403 and 434)

cnmir2 replied:
frank  i will test the druid and give you feed back :)

cnmir2 committed:
rewrite  fix QUEST 27066 27067

FIX THE QUEST 27066 MULGORE priest quest and druid quest 27067

cnmir2 committed:
rewrite the sql and fix training npc

fix  the quest 27066 27067 sql and fix training npc not give training spell option
